src/test/java/RenamePlaylist.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class RenamePlaylist extends BaseTest{
+    @Test
+    public void renamePlaylist() {
+        enterEmail("rumkostia92@gmail.com");
+        enterPassword("te$t$tudent");
+        loginSubmit();
+        enterPlaylist();
+        changePlaylistName();
```

Comment:
Please,use Assertions In public static void renamePlaylist()

For example  `Assert.assertTrue(notificationText());`

```
 public static boolean notificationText() {
        return wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".success"))).isDisplayed();
    }
```



Assert.assertTrue(condition): This method asserts if the condition is true or not. If not, then the exception error is thrown.